presto-main/src/main/java/com/facebook/presto/tdigest/docs/tdigest.graffle

```diff
@@ -25,10 +25,6 @@
 	<string>NO</string>
 	<key>MasterSheets</key>
 	<array/>
-	<key>ModificationDate</key>
```

Comment:
I don't really know what these metadata means. Curious why are we changing it?

Reply:
I had used a file as template that was previously made by Martin Traverso, so he appears to be the creator, and then I appeared as someone who modified the file. Changed per previous comment by Tim.

> There's some copy paste in the graffle metadata. Other than that I think this is good to merge @rongrong